src/email-sender/email-sender.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Example email sending lambda function that works with events.kleros.io.
+ * Replace the email lookup with your subscribers for an event and replace your sendgrid template.
+ */
+const sgMail = require('@sendgrid/mail')
+const Web3 = require('web3')
```

Comment:
I don't see why that is necessary? What if a project wants to use a different network on staging? It seems limiting to me. And I couldn't tell for sure what the network was since its encrypted in the secrets file, which also seems unnecessary

Reply:
That Kovan endpoint will soon be deprecated or severely limited. Infura is becoming a paid service that we will have to pay quotas for. Lambda dashboard env vars are not secure according to their docs and they are meant to be used for configuration variables not API keys.

Yes you can use KMS. https://www.npmjs.com/package/serverless-kms-secrets

We can still encrypt multiple keys if that is desired.

Reply:
AWS encrypts env vars by default. I don't see anything about it being insecure to store API keys there but maybe you are right